tests/test_outputs/test_intram_by_pg.py

```diff
@@ -1,178 +1,210 @@
 """Tests for intram_by_pg.py."""
 # Standard Library Imports
-import pytest
-import os
-import logging
-import pathlib
 
-# Third Party Imports
+from pandas.testing import assert_frame_equal
+import logging
 import pandas as pd
+import unittest
 
 # Local Imports
 from src.outputs.intram_by_pg import output_intram_by_pg
-from tests.test_outputs.conftest import read_config
-
-
-# Assign config values to paths
-config = read_config()
-LOCATION = config["global"]["platform"]
+from src.outputs.intram_by_pg import _generate_intarm_by_pg
 
 # create logger (required pass to function)
 TestLogger = logging.getLogger(__name__)
 
 
-class TestOutputIntramByPG(object):
-    """Tests for output_intram_by_pg."""
+class TestIngramBYPg(unittest.TestCase):
+    """Unit tests for:
+    - intram_by_pg._output_intram_by_pg
+    - intram_by_pg.generate_intarm_by_pg.
+    """
+    def create_config_dict(self):
+        """Creates config dictionary for test"""
+        config = {"outputs_paths": {"outputs_master": "test_path"},
+                  "years": {"survey_year": "2023"}}
+        return config
+    
+    def mock_write_csv(self, filepath: str, data: pd.DataFrame) -> None:
+        """Dummy script mimicking `s3_mods.rc_write_csv`.
```

Comment:
Strictly speaking, the pipeline is platform agnostic, so it's not just the s3_mods func that we're mocking here, it's the function passed to the module as write_csv

Reply:
totally correct. I have amended the comment.